src/Vector.js

```diff
@@ -49,37 +48,22 @@ export default class Vector {
 
     
 
-    calcAngle=(x,y)=>{
-        if(x>0 && y>0){
-            return Math.atan(y/x);
-        }
-        else if(x<0 && y> 0){
-            return Math.PI - Math.atan(y/(-1*x));
-        }
-        else if(x<0 && y<0){
-            return Math.PI + Math.atan(y/x);
-        }
-        else if(x>0 && y<0){
-            return 2*Math.PI - Math.atan(-y/x);
-        }
-        else if(x===0){
-            if(y>0){
-                return Math.PI/2;
-            }
-            else if (y<0){
-                return 1.5* Math.PI;
-            }
-        }
-
-        else if(y===0){
-            if(x>0){
-                return 0;
-            }
-            else if(x<0){
-                return Math.PI;
-            }
+    function calcAngle(x, y) {
```

Comment:
Overall looks much cleaner than before! But could you change the calcAngle function to arrow syntax please? Otherwise this.calcAngle() in the draw function would throw an error.

Reply:
Done!

Reply:
Thanks looks good!